runtime/js-compute-runtime/builtins/client-info.cpp

```diff
@@ -125,6 +224,11 @@ const JSFunctionSpec ClientInfo::methods[] = {
 const JSPropertySpec ClientInfo::properties[] = {
     JS_PSG("address", address_get, JSPROP_ENUMERATE),
     JS_PSG("geo", geo_get, JSPROP_ENUMERATE),
+    JS_PSG("tlsCipherOpensslName", tls_cipher_openssl_name_get, JSPROP_ENUMERATE),
+    JS_PSG("tlsProtocol", tls_protocol_get, JSPROP_ENUMERATE),
+    JS_PSG("tlsJA3MD5", tls_ja3_md5_get, JSPROP_ENUMERATE),
+    JS_PSG("tlsClientCertificate", tls_client_certificate_get, JSPROP_ENUMERATE),
+    JS_PSG("tlsClientHello", tls_client_hello_get, JSPROP_ENUMERATE),
```

Comment:
These are great!

Is it possible to add them to the `.d.ts` types file so they show up as part of `ClientInfo`?